project.godot

```diff
@@ -113,6 +113,11 @@ g_cheat_editor={
 "events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":80,"unicode":0,"echo":false,"script":null)
  ]
 }
+g_cheat_menu={
+"deadzone": 0.5,
+"events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":92,"unicode":0,"echo":false,"script":null)
```

Comment:
Which key is this set to? I think this should default to F6, as my plan for the F keys is:
- F5 quick save
- F9 quick load
- F10 ingame console
- F11 could be another default binding for toggling fullscreen
- F12 take screenshot

Reply:
Currently it's assigned to the ` key. I'll change it to F6

Reply:
I wouldn't be able to use it, then, as you can't type that on a Finnish keyboard with a single press...

Reply:
Actually, you can't press it on a german keyboard either, but because godot makes some kind of difference between normal letters and special chars, I can press ^ instead (left to the 1 on german keyboards)